src/metabase/server/middleware/session.clj

```diff
@@ -74,6 +74,7 @@
 (s/defmethod set-session-cookie :normal
   [request response {session-uuid :id} :- {:id (s/cond-pre UUID u/uuid-regex), s/Keyword s/Any}]
   (let [response       (wrap-body-if-needed response)
+        is-https       (request.u/https? request)
```

Comment:
This is the ultimate nitpick but we use `?` at the end of boolean variable names in the codebase. I know there are different opinions on whether or not to do that in the Clojure world, but the only thing worse than picking one style and sticking with it is mashing a bunch of different styles together.

Reply:
```suggestion
        is-https?      (request.u/https? request)
```